src/GovUk.Education.ExploreEducationStatistics.Data.Model/Repository/FootnoteRepository.cs

```diff
@@ -78,25 +76,100 @@ public FootnoteRepository(StatisticsDbContext context)
             IEnumerable<Guid> filterItemIds,
             IEnumerable<Guid> indicatorIds)
         {
-            var releaseIdParam = new SqlParameter("releaseId", releaseId);
-            var subjectIdParam = new SqlParameter("subjectId", subjectId);
-            var filterItemListParam = CreateIdListType("filterItemList", filterItemIds);
-            var indicatorListParam = CreateIdListType("indicatorList", indicatorIds);
+            // Query the id's of the filter groups and filters corresponding with the filter item id's
+            // Sanitise all id's to make sure they belong to the subject
+            var filterIdsResult = await _context.FilterItem
+                .Where(filterItem => filterItemIds.Contains(filterItem.Id)
+                                     && filterItem.FilterGroup.Filter.SubjectId == subjectId)
```

Comment:
I thought you'd need to do a bunch of "Include" and "ThenInclude" to get this lookup to work: `filterItem.FilterGroup.Filter.SubjectId`?

But I'm also guessing that no way would the unit tests be working if you didn't have this right :)

Reply:
I don't think you need to because my understanding is that includes alter what is selected, and that you would only need them if your referencing attributes of whatever your including after you've executed the query in memory.

EF core seems capable of translating what's required into SQL without them, and the only thing we return after the result is executed are the columns of Footnotes, so we don't need to include them.

Like you say, the unit tests should also cover us if they were needed and I've missed something.

Reply:
Yup, happy that ti would spit out plenty of errors if it wasn't happy by this stage!